runtime/shiden/src/lib.rs

```diff
@@ -1259,16 +1264,53 @@ pub type Executive = frame_executive::Executive<
     Migrations,
 >;
 
+pub struct StaticTierParamsMigration;
+impl OnRuntimeUpgrade for StaticTierParamsMigration {
```

Comment:
I hadn't done this in the past, but Ermal started with it - I'd suggest to use `VersionedMigration` to ensure version consistency.

Reply:
yes good idea! it was not so difficult to provide a configurable paramater type for each runtime